app/src/main/java/org/sopt/android_hyorim_30th/ui/signup/SignUpActivity.kt

```diff
@@ -0,0 +1,44 @@
+package org.sopt.android_hyorim_30th.ui.signup
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import org.sopt.android_hyorim_30th.R
+import org.sopt.android_hyorim_30th.databinding.ActivitySignUpBinding
+import org.sopt.android_hyorim_30th.ui.signin.SignInActivity
+import org.sopt.android_hyorim_30th.util.shortToast
+
+class SignUpActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySignUpBinding
+    private val signUpViewModel: SignUpViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        initViewModel()
+    }
+
+    private fun initViewModel() {
+        binding.viewModel = signUpViewModel
+        binding.lifecycleOwner = this
+    }
+
+    fun initSignUpCompleteBtnClick(view: View) {
+        if (signUpViewModel.isInputComplete) {
+            Intent(this, SignInActivity::class.java).apply {
+                putExtra(KEY_ID, signUpViewModel.userId.value)
+                putExtra(KEY_PW, signUpViewModel.userPw.value)
+                setResult(RESULT_OK, this)
+            }
+            if (!isFinishing) finish()
```

Comment:
그.. 정말 솔직하게 말씀드리면 YB때 암것두 모를때 코드 구글링해서 찾아왔던 건데,
저도 궁금해서 찾아보니 아래와 같은 상황에서 사용한다구 하네요..!
해당 코드에서는 지워도 상관 없을 듯 합니다! ㅎㅎ 

> Check to see whether this activity is in the process of finishing, either because you called [finish()](https://developer.android.com/reference/android/app/Activity#finish()) on it or someone else has requested that it finished. This is often used in [onPause()](https://developer.android.com/reference/android/app/Activity#onPause()) to determine whether the activity is simply pausing or completely finishing. [Android Developer]